fixpatches/build.gradle.kts

```diff
@@ -16,10 +16,14 @@ kotlin {
                 implementation(libs.sarif4k)
                 implementation(libs.multiplatform.diff)
                 implementation(libs.kotlin.logging)
-                implementation(libs.log4j.slf4j2.impl)
                 implementation(libs.log4j.core)
```

Comment:
please move `libs.log4j.core` to test too

Reply:
Sure, moved